<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="matDisplay" name="Mat_Display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>551</width>
      <height>261</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>100000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 120, 120);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_position_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 63, 63);
background-color: rgb(231, 236, 227);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Point</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_P1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P1 (Not set)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_P2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P2 (Not set)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_P3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P3 (Not Set)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_P4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>P4 (Not set)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_transformed">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transformed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_generateTransMat">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Matrix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>matDisplay</class>
   <extends>QLabel</extends>
   <header location="global">matdisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
